### DISCUSSION TITLE
Error baseline generation

### DISCUSSION THREAD

**Discussion by gms8994** [oxc-project/oxc#1691] [Feature Request]
Something that would be great for existing projects is some kind of baseline functionality. For example, another tool (phpstan) [can generate a baseline](https://phpstan.org/user-guide/baseline) that allows those specific errors in those specific files to be ignored, allowing you to back in to a clean codebase.

If oxlint had something like this, it would make adding it to CI/CD pipelines for existing projects much easier, and would make me much more able to add it to my projects.

Any thoughts on this would be welcome!

**Comment by leettaylor** [comment]
Eslint also has this ([Bulk Suppressions](https://eslint.org/docs/latest/use/suppressions)). Would be a great feature to have, it's the one thing holding me back from migratiing.
